### ISSUE TITLE
E_NOTICE level error when updating additional owners.

### ISSUE THREAD

**Issue by karlr-stripe** [stripe/stripe-php#521]
I was following the instructions at https://stripe.com/docs/connect/identity-verification-api#additional-owners to update a connected account with a second additional owner. I noticed that the code actually raises a `Undefined offset: 1` 'error'. With default PHP settings, this is just a notice, and it isn't even logged. 

The error seems to come from `serializeAdditionalOwners`:

https://github.com/stripe/stripe-php/blob/b07887090c13674912a9a1a8fed8e42f174eeab8/lib/Account.php#L229-L237

`$additionalOwners` is the new value being passed in(which may be larger than the existing `$originalValue`), so we basically rely on being able to set an array property at an index that doesn't currently exist(`$originalValue[$i]`). This works fine generally since it only throws an E_NOTICE level error.

However, if you have set up your PHP environment to convert errors into fatal exceptions(http://php.net/manual/en/class.errorexception.php#errorexception.examples), this can be a problem. I'm not sure how to proceed here since I'm not very familiar with PHP best practises — is it a problem that we generate an E_NOTICE level error and something that we should fix?

I have a repro script here : https://gist.github.com/karlr-stripe/a9447ef36d688afd84a5c1f3f9b294cf 

You can workaround this by updating the account object this way instead : 

```
$count = count($stripeAccount->legal_entity->additional_owners);
$newAdditionalOwner = [
    $count => [
      'first_name' => 'Jane',
      'last_name' => 'Smith',
    ]
  ];

  \Stripe\Account::update($stripeAccount->id, array(
    "legal_entity" => array(
      "additional_owners" => $newAdditionalOwner
    )
  ));
```

**Comment by ob-stripe** [comment]
Hey @karlr-stripe, thanks for the report.

> is it a problem that we generate an E_NOTICE level error and something that we should fix?

Yep, definitely.

This is probably my bad -- I ported the serialization logic over from stripe-ruby a while back, but Ruby simply returns a null value when trying to access invalid keys in a hash, without any warnings. I'll work on a fix.

**Comment by ob-stripe** [comment]
Fixed in 6.17.2.
